montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -23,6 +23,10 @@
 import montblanc
 import montblanc.impl.common.loaders
 
+ORDER_CASA = 'casa'
+ORDER_OTHER = 'other'
+VALID_ORDERINGS = [ORDER_CASA, ORDER_OTHER]
```

Comment:
I'm not familiar with the A x B notation but Measurement Set rows in CASA are ordered by going through each baseline for a single timeslot and then moving on to the next time entry.